### PR TITLE
new traffic icon style

### PR DESCRIPTION
**Pull request by breakwa11**

### CODE REVIEW

**Comment by wongsyrone** [comment]
LGTM. Any opinion on this?

**Comment by Noisyfox** [comment]
Great.

**Comment by wongsyrone** [comment]
@celeron533 What about yours?

File: shadowsocks-csharp/View/LogForm.Designer.cs [hunk 1]
```diff
@@ -268,11 +268,13 @@ private void InitializeComponent()
             this.trafficChart.Palette = System.Windows.Forms.DataVisualization.Charting.ChartColorPalette.None;
             series1.ChartArea = "ChartArea1";
             series1.ChartType = System.Windows.Forms.DataVisualization.Charting.SeriesChartType.Line;
+            series1.Color = System.Drawing.Color.FromArgb(((int)(((byte)(255)))), ((int)(((byte)(128)))), ((int)(((byte)(0)))));
```
Comment (wongsyrone) [inline shadowsocks-csharp/View/LogForm.Designer.cs]: It seems the type casting is redundant.
ref: https://msdn.microsoft.com/en-us/library/cce5h557(v=vs.110).aspx

Comment (celeron533) [inline shadowsocks-csharp/View/LogForm.Designer.cs]: Agree.

**Pull request merged**